### PR TITLE
Add host CI build

### PR DESCRIPTION
**Pull request by BorjaOuterelo**

### CODE REVIEW

File: micro_ros_setup/scripts/build_firmware.sh [hunk 1]
```diff
@@ -33,7 +33,12 @@ fi
 . $(dirname $0)/clean_env.sh
 
 # source dev_ws
-DEV_WS_DIR=$FW_TARGETDIR/dev_ws
+if [ $RTOS != "host" ]; then
```
Comment (iluetkeb) [inline micro_ros_setup/scripts/build_firmware.sh]: Maybe we can simply create an empty dev_ws for host builds, thus saving this kind of conditional.

File: micro_ros_setup/scripts/create_firmware_ws.sh [hunk 2]
```diff
@@ -84,7 +85,11 @@ fi
 
 # build the dev_ws
 . $(dirname $0)/clean_env.sh
-pushd $FW_TARGETDIR/$DEV_WS_DIR >/dev/null
+if [ $RTOS != "host" ]; then
```
Comment (iluetkeb) [inline micro_ros_setup/scripts/create_firmware_ws.sh]: I think this is wrong. For the host, the dev_ws build should just be empty, so building does nothing.

If you instead build the entire firmware folder, that totally changes what create_firmware_ws.sh does.

File: micro_ros_setup/scripts/build_firmware.sh (hunk 1 above)
Comment (BorjaOuterelo) [reply micro_ros_setup/scripts/build_firmware.sh]: Already created in the last commits.

File: micro_ros_setup/scripts/create_firmware_ws.sh (hunk 2 above)
Comment (BorjaOuterelo) [reply micro_ros_setup/scripts/create_firmware_ws.sh]: True, solved in the last commits.

**Pull request merged**